fwb/app/api/discounts/route.ts

```diff
@@ -2,22 +2,10 @@ import { NextRequest, NextResponse } from "next/server";
 import { auth, currentUser } from "@clerk/nextjs";
 import supabaseClient from "@/supabase";
 
-interface Discount {
```

Comment:
We should definetly add typing/validation to our API in the future but I think we can forego that for now.

Reply:
Yes I agree. I originally planned for an interface but as I wrote the code I didn't utilize it much since I changed the types of the columns in supabase often.